src/Components/Shared/Button/button.module.css

```diff
@@ -0,0 +1,65 @@
+.delete {
+  margin: 0 5px;
+  padding: 2px 10px 0px 10px;
+  min-height: 30px;
+  border-radius: 5px;
+  font-weight: bold;
+  text-align: center;
+  cursor: pointer;
+  background-color: #a30000;
+  border: 1.5px solid #ffffff;
+  color: #ffffff;
+}
+
+.edit {
+  margin: 0 5px;
+  padding: 2px 10px 0px 10px;
+  min-height: 30px;
+  border-radius: 5px;
+  font-weight: bold;
+  text-align: center;
+  cursor: pointer;
+  background-color: #9464c4;
+  border: 1.5px solid #ffffff;
+  color: #000000;
+}
+
+.add {
+  margin: 0 5px;
+  padding: 2px 10px 0px 10px;
+  min-height: 30px;
+  border-radius: 5px;
+  font-weight: bold;
+  text-align: center;
+  cursor: pointer;
+  background-color: #00a34c;
+  border: 1.5px solid #ffffff;
+  color: #ffffff;
+}
+
+.disabled {
+  margin: 0 5px;
+  padding: 2px 10px 0px 10px;
+  min-height: 30px;
+  border-radius: 5px;
+  font-weight: bold;
+  text-align: center;
+  opacity: 0.5;
+  cursor: not-allowed;
+  background-color: #cccccc;
+  border: 1.5px solid #cccccc;
+  color: #666666;
+}
+
+.white {
+  margin: 0 5px;
+  padding: 2px 10px 0px 10px;
+  min-height: 30px;
+  border-radius: 5px;
+  font-weight: bold;
+  text-align: center;
+  cursor: pointer;
+  background-color: #ffffff;
+  border: 1.5px solid #ffffff;
+  color: #000000;
+}
```

Comment:
To be able to reuse some code, you could create a class "btn" that already includes the repeated elements in all buttons and use the rest of the classes to style what makes them different.